lib/connect.js

```diff
@@ -303,9 +303,6 @@ function connect(token, options) {
     isNonArrayObject(options.networkQuality) ? options.networkQuality : {}
   );
 
-  // Convert options.networkQuality to boolean to configure Media Signaling
-  options.networkQuality = isNonArrayObject(options.networkQuality) || options.networkQuality;
```

Comment:
@makarandp0 Won't removing this line screw up MSP negotiation for network quality?

Reply:
no, because this is now moved to  https://github.com/twilio/twilio-video.js/pull/1308/files#diff-6d7c816017e979b03f29300b8b413bd5dd986fe4090da649535b21047c28ddbbR144